src/main/kotlin/dev/jbang/idea/JBang.kt

```diff
@@ -55,7 +55,15 @@ fun getJBangCmdAbsolutionPath(): String {
     }
 
     if(actualJBangScript==null) {
+        // try ~/.jbang/bin/jbang
         actualJBangScript = Path.of(userHome, ".jbang/bin/$jbangScript")
+        if (!Files.exists(actualJBangScript)) {
+            // try ~/.sdkman/candidates/jbang/current/bin/jbang
+            val sdkmanJBangScript = Path.of(userHome, ".sdkman/candidates/jbang/current/bin/$jbangScript")
```

Comment:
I'm not a fan of this. We search jbang home from settings, path and ~/.jbang/bin/ 

It should not be necessary to add sdkman since we would then need to also add mise, homebrew, etc.

Reply:
Anybody installing IntelliJ as a snap package on Linux will run into this issue. Although jbang is in the PATH on the command-line, it is not in the PATH as seen by IntelliJ (restricted snap runtime environment).

This simple fix adds a lot of value to end-users.

Reply:
Also look in settings. There is a setting to set the explicit jbang to use.